README.md

```diff
@@ -73,17 +73,17 @@ The `core` directory contains the three core libraries: `lib`, `codegen`, and
 `http` published as `rocket`, `rocket_codegen` and `rocket_http`, respectively.
 The latter two are implementations details and are reexported from `rocket`.
 
-### Testing
+## Testing
```

Comment:
This changes the organization of the README. It's confusing that there are both cosmetic and functional changes in the same commit.

Reply:
Oops, sorry, it was a typo. I'm going to fix it...

Reply:
Done.